stepper.browser.json

```diff
@@ -1,5 +1,3 @@
 {
-    "dependencies": [
-         "require: ./stepper"
-    ]
+    "dependencies": ["require: ./stepper"]
```

Comment:
Hmm, don't all these json changes go against our eslint rules? I'm pretty sure our coding standard is to not have array blocks declared on the same line like this.

Reply:
So the advantage of prettier is that it takes care of styling like this for us, so we don't have to worry about it. 
The `eslint-config-prettier` purposely turns off styling rules like that, so we don't have to discuss code styling during code reviews, etc.

Of course the disadvantage is that sometimes the styling may not be what we're always used to. Let me look and see if this is a configurable option for prettier to have array blocks declared on separate lines as we have before this proposed change.